bindings/capacitorjs/src/types/account.ts

```diff
@@ -0,0 +1,146 @@
+import type { Address, AddressWithUnspentOutputs } from './address';
```

Comment:
Maybe a stupid question because I haven't looked into how it is build. Do we really need to have all the type files duplicated or could they also just reference the type files from the nodejs binding directly?

Reply:
Super valid 👌 in the beginning was trying use them from the nodejs folder, but to build the package with Rollup at `npm build` are needed inside the /src folder. We could copy them from nodejs when install dependencies probably.
As this version is not paired with nodejs but with latest Java CI build, by now we could keep it, but will do a new PR with the updates to the latest state and will copy them in some way, avoiding duplicating code

Also there are some changes in some files adapted to the Capacitor JS plugin API, mostly at `bindings.ts`, but I think we can address it in the future PR

Reply:
Ok for now, but good that you keep it in mind :+1: